Python/getargs.c

```diff
@@ -1705,8 +1705,10 @@ vgetargskeywords(PyObject *args, PyObject *kwargs, const char *format,
             }
             if (max < nargs) {
                 PyErr_Format(PyExc_TypeError,
-                             "Function takes %s %d positional arguments"
+                             "%.200s%s takes %s %d positional arguments"
                              " (%d given)",
+                             (fname == NULL) ? "function" : fname,
```

Comment:
The reason was: because @serhiy-storchaka [said so](https://bugs.python.org/msg290896).